src/main/java/org/apache/sling/resourceresolver/impl/ResourceResolverFactoryActivator.java

```diff
@@ -313,10 +314,11 @@ protected void activate(final BundleContext bundleContext, final ResourceResolve
                 String value = prefix.trim();
                 if (!value.isEmpty()) {
                     if (value.startsWith("/")) { // absolute path should be given
+                        // path must not end with "/" to be valid absolute path
                         if (value.endsWith("/")) {
-                            prefixSet.add(value);
+                            prefixSet.add(StringUtils.removeEnd(value, "/"));
```

Comment:
`StringUtils.removeEnd` already has a check so you can remove the surrounding if